core-build/gulp-core-build/src/tasks/GulpTask.ts

```diff
@@ -277,11 +277,9 @@ export abstract class GulpTask<TTaskConfig> implements IExecutable {
             (file: gutil.File,
               encoding: string,
               callback: (p?: Object) => void) => {
-                'use strict';
```

Comment:
>'use strict'; [](start = 16, length = 13)

why do you remove it?

Reply:
It throws an exception. For example, you can't use `...` or default values in functions if you add "use strict"